### PR TITLE
OBG API clean up

### PR DESCRIPTION
**Pull request by tsehelnyk**

### CODE REVIEW

File: opba-banking-rest-api-ymls/src/main/resources/static/tpp_banking_api_pis.yml [hunk 1]
```diff
@@ -191,13 +182,6 @@ components:
         instantPayment:
           type: boolean
 
-    remittanceInformationUnstructured:
```
Comment (valb3r) [inline opba-banking-rest-api-ymls/src/main/resources/static/tpp_banking_api_pis.yml:194]: Please keep this field

File: opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml [hunk 2]
```diff
@@ -185,7 +147,7 @@ paths:
       security:
         - sessionCookie: []
 
-  /v1/consent/{auth-id}/fromAspsp/{redirectState}/ok/{fromAspspRedirectCode}:
+  /v1/consent/{auth-id}/fromAspsp/ok/{redirectState}:
```
Comment (valb3r) [inline opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml:150]: Suggest to rephrase endpoint to:
`/v1/consent/{auth-id}/fromAspsp/ok/{redirectState}` -> `/v1/consent/{auth-id}/fromAspsp/{redirectState}/ok

File: opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml [hunk 3]
```diff
@@ -226,7 +187,7 @@ paths:
       security:
         - redirectCookie: []
 
-  /v1/consent/{auth-id}/fromAspsp/{redirectState}/nok/{fromAspspRedirectCode}:
+  /v1/consent/{auth-id}/fromAspsp/nok/{redirectState}:
```
Comment (valb3r) [inline opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml:190]: Suggest to rephrase endpoint to:
`/v1/consent/{auth-id}/fromAspsp/nok/{redirectState}` -> `/v1/consent/{auth-id}/fromAspsp/{redirectState}/nok`

File: opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml [hunk 4]
```diff
@@ -348,17 +307,17 @@ components:
       schema:
         type: string
 
-    redirectCode:
+    X-XSRF-TOKEN:
       name: redirectCode
```
Comment (valb3r) [inline opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml:311]: Can we use same name and declaration please, it is confusing to have redirectCode and `X-XSRF-TOKEN` both

File: opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml [hunk 5]
```diff
@@ -272,10 +232,9 @@ components:
         format: uuid
 
     X-XSRF-TOKEN:
+      name: Redirect-Code
```
Comment (valb3r) [inline opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml:235]: Can we use same name and declaration please, it is confusing to have redirectCode and `X-XSRF-TOKEN` both

File: opba-banking-rest-api-ymls/src/main/resources/static/tpp_banking_api_pis.yml (hunk 1 above)
Comment (tsehelnyk) [reply opba-banking-rest-api-ymls/src/main/resources/static/tpp_banking_api_pis.yml:194]: ok

File: opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml (hunk 2 above)
Comment (tsehelnyk) [reply opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml:150]: endpoint rephrased

File: opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml (hunk 3 above)
Comment (tsehelnyk) [reply opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml:190]: endpoint rephrased

File: opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml (hunk 5 above)
Comment (tsehelnyk) [reply opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml:235]: changed header name to "X-XSRF-TOKEN"

**Review by valb3r — APPROVED** [review]

File: opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml (hunk 4 above)
Comment (tsehelnyk) [reply opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml:311]: changed to xXsrfToken

**Pull request merged**